examples/backend.py

```diff
@@ -0,0 +1,48 @@
+"""
+MPyG321 different backend example
+Playing and pausing some music but using mpg123 as the backend instead
+You need to add a "sample.mp3" file in the working directory
+"""
+
+from mpyg321.mpyg321 import MPyg321Player
+from time import sleep
+
+def do_some_play_pause(player):
+    """Does some play and pause"""
+    player.play_song("sample.mp3")
+    sleep(5)
+    player.pause()
+    sleep(3)
+    player.resume()
+    sleep(5)
+    player.stop()
+
+
+def do_some_jumps(player):
+    """Does some jumps"""
+    player.play_song("sample.mp3")
+    sleep(3)
+    print("Jumping to MPEG frame 200...")
+    player.jump(200)
+    sleep(3)
+    print("Jumping to 1 second...")
+    player.jump("1s")
+    sleep(3)
+    print("Jumping forward 20 MPEG frames...")
+    player.jump("+20")
+    sleep(3)
+    print("Jumping back 1 second...")
+    player.jump("-1s")
+    sleep(3)
+    player.stop()
+
```

Comment:
I think there is a newline needed here for pep8 compliance. Please check your pep8 compliance when you submit some code. There are some great IDE extensions for automatic pep8 checking

Reply:
I've added linebreaks in this file to comply with PEP8. I will look into getting a tool to check for PEP8 violations, thank you for the recommendation!

Reply:
Sorry for the correction here. I think this file is irrelevant now, because the "backend" parameter does not exist any more.

Reply:
I think this file should be removed, doesn't it?

Reply:
Good point. I've removed it.